### PR TITLE
Digitdustin/restyling

### PR DESCRIPTION
**Pull request by digitdustin**
Restyling of entire student-facing portion of Edugator:
- Coding playground
- Landing Page

### CODE REVIEW

File: pages/code/[problemId].tsx [hunk 1]
```diff
@@ -66,44 +77,219 @@ export default function CodeEditor() {
     if (defaultStdin) {
       setStdin(defaultStdin);
     }
-  }, [defaultStdin]);
+    /* if (currentProblem) {
+      setEditorCode(currentProblem?.code?.body);
+    } */
+  }, [currentProblem]);
```
Comment (kapooramanpreet) [inline pages/code/[problemId].tsx:83]: I think this should be `defaultStdin`

**Review by kapooramanpreet — APPROVED** [review]
We can merge once you fix the CI tests.

**Review by kapooramanpreet — APPROVED** [review]
LGTM!

**Review by marcgabe15 — APPROVED** [review]
Tested it out for the most part on my end looks good!

**Pull request merged**